### PR TITLE
[22] Newsletter Form Validation

### PR DESCRIPTION
**Pull request by Cynthia-Berry**
## Task: `[22] Newsletter Form Validation`
- `Added validation to all the newsletter at the footer
newsletter
newsletter2

`

### CODE REVIEW

File: submissions/cynthiaNwakaeme/domManipulation/newsletter.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+// To validate the Newsletter
+sendForm.addEventListener("submit", require = (event) => {
+    const emailRegex = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,10})+$/;
+
+    if (newsletter.value === "") {
+        errorMessage.innerHTML = "Please enter your email";
+        event.preventDefault();
+        return false;
+    } else
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/domManipulation/newsletter.js:13]: The `else if` statement should be on the same line @Cynthia-Berry

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/cynthiaNwakaeme/domManipulation/newsletter.js (hunk 1 above)
Comment (Cynthia-Berry) [reply submissions/cynthiaNwakaeme/domManipulation/newsletter.js]: @ToluAkin  Good Day ma,
I have effected the changes.


Thank you for observing that.
Kindly review my PR.

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Cynthia-Berry there are comments that needs to be attended to

**Comment by Cynthia-Berry** [comment]
> @Cynthia-Berry there are comments that needs to be attended to

@ToluAkin  I have implimented the changes. The `else if``  are on the same line now

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Cynthia-Berry Part of task[40] is in this pull request. You need to remove it to avoid confusion later on.

**Comment by ToluAkin** [comment]
@Cynthia-Berry The automated reviewer left comments on your js file that you need to fix.

**Comment by Cynthia-Berry** [comment]
@ToluAkin  I have taken them the linked js file for task[40]

File: submissions/cynthiaNwakaeme/HTML/index.html [hunk 2]
```diff
@@ -42,23 +42,15 @@ <h2>TARIES EMPIRE</h2>
...
-                            </tr>
-                        </table>
-                    </span>
-                <br><br>
-                <span><em>Rating: </em>&#9733;&#9733;&#9733;&#9733;&#9734;</span>
-                <br><br>
-
-                <button>BUY NOW</button>
+                <h3>J-ROYALTY ORGAINC BLACK SOAP</h3><br>
+                <span id="quantity">
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/HTML/index.html:46]: @Cynthia-Berry This changes should be taken out as  well.

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Cynthia-Berry Have you seen the automated comments?

**Comment by Cynthia-Berry** [comment]
@ToluAkin Yes and I have taken out the code for task[40]

**Comment by Cynthia-Berry** [comment]
It is quite hard to track . What do you you advice I do

**Comment by ToluAkin** [comment]
@Cynthia-Berry Open the files changed tab, copy the previous change made and replace with the new changes.

**Comment by Cynthia-Berry** [comment]
@ToluAkin Thank you.  I've fixed it

**Review by ToluAkin — CHANGES_REQUESTED** [review]
The only comments that needs to be fixed is the automated reviewer's comments.

**Comment by ToluAkin** [comment]
@Cynthia-Berry Please check that all checks have passed from the automated review before requesting for review.

**Pull request closed without merging**